lib/sockets/pbpal_resolv_and_connect_sockets.c

```diff
@@ -119,7 +126,7 @@ static void get_dns_ip(struct pbdns_servers_check* dns_check, struct sockaddr* a
                 || (dns_check->dns_server_check & dns_check->dns_mask)) {
                 dns_check->dns_mask <<= 1;
                 addr->sa_family = AF_INET;
-                inet_pton(AF_INET, PUBNUB_DEFAULT_DNS_SERVER, p);
+                get_default_dns_ip((struct pubnub_ipv4_address*)p);
```

Comment:
This is under the condition `#if PUBNUB_USE_IPV6` . Should it be `pubnub_ipv6_address` instead of `pubnub_ipv4_address`?

Reply:
No, I don't think so. See line 128 above:

```c
addr->sa_family = AF_INET;
```

This sets the address family to IPv4. For another comparison, line 112 sets it to IPv6:

```c
addr->sa_family = AF_INET6;
```